workers/config/src/main/kotlin/ConfigWorker.kt

```diff
@@ -81,8 +81,19 @@ class ConfigWorker(
         )
 
         // TODO: Currently the path to the validation script is hard-coded. It may make sense to have it configurable.
-        val validationScript = configManager.getFileAsString(resolvedJobConfigContext, VALIDATION_SCRIPT_PATH)
+        if (!configManager.containsFile(resolvedJobConfigContext, VALIDATION_SCRIPT_PATH)) {
+            db.dbQuery {
```

Comment:
Would it be more elegant to have just one `dbQuery` statement instead of repeating it? The `if` block would just produce the parameters, the resolved configs, the labels, and the issues. In case no script is available, the original configs and empty labels and issues are returned. This would slightly reduce redundancy.

Reply:
> Would it be more elegant to have just one `dbQuery` statement instead of repeating it?

Not sure if it's really more elegant (as it increases nesting and dealing with destructed variables), but done.